### ISSUE TITLE
Windows Service fails due to timeout issue.

### ISSUE THREAD

**Issue by sirredbeard** [rancher/rancher#33832]
This is a rancher/rancher bug report for https://github.com/rancher/rke2/issues/1467.

The PR with the fix is https://github.com/rancher/rke2/pull/1475

**Environmental Info:**
RKE2 Version: 

rke2.exe version v1.21.3+rke2r1 (2ed0b0d1b6924af4414393cd1796c174a1ff5352)
go version go1.16.6b7

Node(s) CPU architecture, OS, and Version: 

```
WindowsBuildLabEx                                       : 19041.1.amd64fre.vb_release.191206-1406
WindowsCurrentVersion                                   : 6.3
WindowsEditionId                                        : ServerStandardACor
WindowsInstallationType                                 : Server Core
WindowsInstallDateFromRegistry                          : 7/26/2021 2:10:32 PM
WindowsProductId                                        : 00431-30000-00000-AA117
WindowsProductName                                      : Windows Server Standard
```

Cluster Configuration:
1 server
1 windows node

**Describe the bug:**
Running the agent as a windows service errors with a timeout issue, why executing the agent not as a service works.

```
rke2.exe agent service --add
```

**Steps To Reproduce:**
Install RKE2 for Windows and try to run the agent as a windows service.

**Expected behavior:**
rke2 starts up as a windows service and joins a cluster.

**Actual behavior:**
rke2 starts then fails.

**Comment by deniseschannon** [comment]
Available as part of RKE2 v1.21.3-rc1+rke2e2

**Comment by rancher-max** [comment]
Confirmed this is working properly in RKE2 issue.
